Artifacts/windows-create-url-shortcut/CreateUrlShortcut.ps1

```diff
@@ -0,0 +1,216 @@
+<##################################################################################################
+
+    Description
+    ===========
+
+    - This script creates a .url (web) shortcut according to user's specifications. 
+
+    - The following logs are generated on the machine - 
+        - This script's log : $PSScriptRoot\url-shortcut-creator folder.
+
+
+    Prerequisite
+    ============
+
+    - Ensure that the powershell execution policy is set to unrestricted or bypass.
+
+    - Ensure that powershell is run elevated.
+
+
+    Known issues / Caveats
+    ======================
+    
+    - No known issues.
+
+
+    Coming soon / planned work
+    ==========================
+    
+    - N/A.
+
+##################################################################################################>
+
+#
+# Optional arguments to this script file.
+#
+
+Param(
+    # 
+    [ValidateNotNullOrEmpty()]
+    $ShortcutName,
+
+    # 
+    [ValidateNotNullOrEmpty()]
+    $ShortcutTargetPath
+)
+
+##################################################################################################
+
+#
+# Powershell Configurations
+#
+
+# Note: Because the $ErrorActionPreference is "Stop", this script will stop on first failure.  
+$ErrorActionPreference = "stop"
+
+###################################################################################################
+
+#
+# Custom Configurations
+#
+
+# Location of the log files
+$ShortcutCreatorFolder = Join-Path $PSScriptRoot -ChildPath $("Url-Shortcut-Creator-" + [System.DateTime]::Now.ToString("yyyy-MM-dd-HH-mm-ss"))
+$ScriptLogFolder = Join-Path -Path $ShortcutCreatorFolder -ChildPath "Logs"
+$ScriptLog = Join-Path -Path $ScriptLogFolder -ChildPath "ShortcutCreator.log"
+
+##################################################################################################
+
+# 
+# Description:
+#  - Displays the script argument values (default or user-supplied).
+#
+# Parameters:
+#  - N/A.
+#
+# Return:
+#  - N/A.
+#
+# Notes:
+#  - Please ensure that the Initialize() method has been called at least once before this 

```

Comment:
No. Not designed to be idempotent. Just a simple private function (not a publicly exposed cmdlet).